src/components/Table/StudentsTable.tsx

```diff
@@ -62,36 +24,44 @@ const StudentsTable: FC = () => {
     date: '',
   });
 
-  // Handle sorting
+  const { formattedData, loading, error } = useFormattedStudentData();
+  //to fix
```

Comment:
Kdyz chces nejake veci menit v budoucnosti doporucuji začínat koment s TODO. Pak IDE ti to umí zvýraznit, vyspsat všechny todos atd.